<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3AxisAcc3D</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>574</width>
      <height>564</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>24</width>
      <height>561</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: none; /* Kenarlığı kaldırır */
    background-color: transparent; /* Arkaplan rengini transparan yapar */
}

QProgressBar::chunk {
    background-color: red; /* İlerleme çizgilerinin başlangıç rengi (siyah) */
    width: 10px; /* Her bir ilerleme noktasının genişliği */
    margin: 5px; /* Noktalar arasındaki boşluk */
	height: 4px;
}</string>
    </property>
    <property name="maximum">
     <number>13320</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>574</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="layoutLabelRot">
         <item>
          <widget class="QLabel" name="labelxRot">
           <property name="text">
            <string>xRot : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelyRot">
           <property name="text">
            <string>yRot :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelzRot">
           <property name="text">
            <string>zRot :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutRot">
         <item>
          <widget class="QLabel" name="xRot">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="yRot">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="zRot">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="layoutLabelRotCam">
         <item>
          <widget class="QLabel" name="labelxRotCam">
           <property name="text">
            <string>xRotCam : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelyRotCam">
           <property name="text">
            <string>yRotCam : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelzRotCam">
           <property name="text">
            <string>zRotCam : </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutRotCam">
         <item>
          <widget class="QLabel" name="xRotCam">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="yRotCam">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="zRotCam">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="layoutLabelAngle">
         <item>
          <widget class="QLabel" name="labelx">
           <property name="text">
            <string>x : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labely">
           <property name="text">
            <string>y : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelz">
           <property name="text">
            <string>z : </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutAngle">
         <item>
          <widget class="QLabel" name="xAngle">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="yAngle">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="zAngle">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>84</width>
      <height>636</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layoutQuit">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Reset Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Start Fly</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Turn X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Turn Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>Turn Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderZ">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>24</width>
      <height>561</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: none; /* Kenarlığı kaldırır */
    background-color: transparent; /* Arkaplan rengini transparan yapar */
}

QProgressBar::chunk {
    background-color: black; /* İlerleme çizgilerinin başlangıç rengi (siyah) */
    width: 10px; /* Her bir ilerleme noktasının genişliği */
    margin: 5px; /* Noktalar arasındaki boşluk */
	height: 4px;
}</string>
    </property>
    <property name="maximum">
     <number>13120</number>
    </property>
    <property name="value">
     <number>13120</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>436</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3000ft</string>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>450</y>
      <width>70</width>
      <height>3</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>6000ft</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>324</y>
      <width>70</width>
      <height>3</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>198</y>
      <width>70</width>
      <height>3</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>184</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>9000ft</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>72</y>
      <width>70</width>
      <height>3</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>58</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>12000ft</string>
    </property>
   </widget>
   <zorder>line_2</zorder>
   <zorder>line_1</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>progressBar_2</zorder>
   <zorder>progressBar</zorder>
   <zorder>widget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_9</zorder>
   <zorder>label_12</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>776</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
